packages/lib/src/vis/shared/AxisSystemProvider.tsx

```diff
@@ -37,8 +37,27 @@ function AxisSystemProvider(props: PropsWithChildren<Props>) {
   const dataToWorld = (vec: Vector2 | Vector3) =>
     new Vector2(abscissaScale(vec.x), ordinateScale(vec.y));
 
+  const { width, height } = availableSize;
+  const cameraToHtmlMatrix = new Matrix4()
+    .makeScale(width / 2, -height / 2, 1)
+    .setPosition(width / 2, height / 2); // account for shift of (0,0) position (center for camera, top-left for HTML)
+
   const [interactionMap] = useState(new Map<string, Interaction>());
 
+  const registerInteraction = useCallback(
+    (id: string, value: Interaction) => {
+      interactionMap.set(id, value);
+    },
+    [interactionMap]
+  );
+
+  const unregisterInteraction = useCallback(
+    (id: string) => {
+      interactionMap.delete(id);
+    },
+    [interactionMap]
+  );
+
```

Comment:
Just moving `shouldInteract` below the other two functions, to match the order in which the API is used.